src/app/visual/progress-indicator/progress-indicator-visual.component.html

```diff
@@ -1,37 +1,251 @@
+      </button>
+    </sky-progress-indicator-item>
+
+    <sky-progress-indicator-item
+      title="Test connection"
+    >
+      <button
+        class="sky-btn sky-btn-primary"
+        type="button"
+      >
+        Test connection
+      </button>
+    </sky-progress-indicator-item>
+
+    <sky-progress-indicator-item
+      title="Turn on connection"
+    >
+      Content 4
+    </sky-progress-indicator-item>
+
+    <sky-progress-indicator-nav-button
+      buttonType="reset"
+      buttonText="Erase all settings"
+    >
+    </sky-progress-indicator-nav-button>
+  </sky-progress-indicator>
+
+  <h3>
+    Interact with the progress indicator programmatically
+  </h3>
+
+  <p>
+    <button
+      class="sky-btn"
+      type="button"
+      (click)="onPreviousClick()"
+    >
+      Previous
+    </button>
+
+    <button
+      class="sky-btn sky-btn-primary"
+      id="app-screenshot-display-mode-vertical-next-button"
+      type="button"
+      (click)="onNextClick()"
+    >
+      Next
+    </button>
+
+    <button
+      class="sky-btn"
+      type="button"
+      (click)="onGoToClick()"
+    >
+      Goto first
+    </button>
+  </p>
+</div>
+
+<h2>
+  Horizontal mode
+</h2>
+
+<div
+  class="app-screenshot"
+  id="app-screenshot-display-mode-horizontal"
+>
+  <sky-progress-indicator
+    displayMode="horizontal"
+    [messageStream]="messageStreamHorizontal"
+    (progressChanges)="onProgressChanges($event)"
+    #horizontalIndicator
+  >
+    <sky-progress-indicator-item
+      title="Configure connection"
+    >
+      Content 1
+    </sky-progress-indicator-item>
+
+    <sky-progress-indicator-item
+      title="Select remote server"
+    >
+      Content 2
+    </sky-progress-indicator-item>
+
+    <sky-progress-indicator-item
+      title="Test connection"
+    >
+      Content 3
+    </sky-progress-indicator-item>
 
-  <sky-progress-indicator-item
-    title="Do the first thing"
+    <sky-progress-indicator-item
+      title="Turn on connection"
+    >
+      Content 4
+    </sky-progress-indicator-item>
+  </sky-progress-indicator>
+
+  <sky-progress-indicator-nav-button
+    buttonType="previous"
+    [disabled]="disabled"
+    [progressIndicator]="horizontalIndicator"
+  >
+  </sky-progress-indicator-nav-button>
+
+  <sky-progress-indicator-nav-button
+    buttonType="next"
+    id="app-screenshot-display-mode-horizontal-next-button"
+    [disabled]="disabled"
+    [progressIndicator]="horizontalIndicator"
   >
-  </sky-progress-indicator-item>
+  </sky-progress-indicator-nav-button>
 
-  <sky-progress-indicator-item
-    title="Do the second thing"
+  <sky-progress-indicator-nav-button
+    buttonType="finish"
+    [disabled]="disabled"
+    [progressIndicator]="horizontalIndicator"
   >
-  </sky-progress-indicator-item>
+  </sky-progress-indicator-nav-button>
 
-  <sky-progress-indicator-item
-    title="Do the third thing"
+  <sky-progress-indicator-nav-button
+    buttonType="reset"
+    buttonText="Start over"
```

Comment:
Good catch.

Reply:
`buttonText` should be before `buttonType`.